mailbag/derivatives/pdf.py

```diff
@@ -0,0 +1,43 @@
+import os
+from subprocess import Popen, PIPE
+from mailbag.derivative import Derivative
+
+
+class ExampleDerivative(Derivative):
+    derivative_name = 'pdf'
+
+    def __init__(self, email_account, **kwargs):
+        print("Setup account")
+        super()
+
+    def do_task_per_account(self):
+        print(self.account.account_data())
+
+    def do_task_per_message(self, message, args):
+        wkhtmltopdf = 'wkhtmltopdf.exe'
+        html = message.HTML_Body
+        From_name=message.From.split("<")
+        To_name=message.To.split("<")
+        #htmlnew="<table>  <tr>    <th>From</th>    <th>To</th> </tr>"  + "<tr> <td>"+a+d[1][:-1]+"</td>"+"<td>"+b+c[1][:-1]+"</td>  </tr>"+"</table>"
+        #Adding From and To fields in pdf
+        html_from_to="From :"+"&nbsp"+ From_name[0]+ From_name[1][:-1]+"<br>"+"To:"+"&nbsp"+To_name[0]+To_name[1][:-1]
+        html_name = os.path.join(args.directory, args.mailbag_name,str(message.Mailbag_Message_ID )+".html")
+        pdf_name =os.path.join(args.directory, args.mailbag_name,str(message.Mailbag_Message_ID )+".pdf")
+        write_html = open(html_name, 'w')
```

Comment:
This is going to fail with `--dry-run`. Make sure to log what's being written and then only right if dry run is not set.